src/components/Header/Nav/Links/index.js

```diff
@@ -14,23 +13,19 @@ const Links = styled('div')({
 });
 Links.displayName = 'Links';
 
-export default props => (
+export default connect('header')(({ header, ...props }) => (
```

Comment:
What is the significance of `'header'` here? The unistore docs are a bit scant on the details!

Reply:
yeah took me a while - it's the key you're grabbing from the store

Reply:
Yeah, I get you. So the developer would have to know what the data from the store looks like?